schedule/yast/usb_mediacheck.yaml

```diff
@@ -0,0 +1,6 @@
+---
+name: USBmediacheck
```

Comment:
we normally keep the name of the file (or something similar) just in case you want to follow that pattern:
`usb_mediacheck`

Reply:
done